### PR TITLE
feat(changelog): kubernetes-removed-test 2023-02-07

### PR DESCRIPTION
**Pull request by ofranc**
Reporter: Benedikt Rollik

This PR is a new changelog contribution. It has been created automatically.

Make sure to review it carefully before merging it.

### CODE REVIEW

File: changelog/february2023/2023-02-07-kubernetes-removed-test.mdx [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+status: removed
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-02-07
+category: compute
+product: kubernetes
+---
+
+From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with cilium or `calico` as CNI.
```
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-test.mdx:12]:
```suggestion
From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with `cilium` or `calico` as CNI. 
```

File: changelog/february2023/2023-02-07-kubernetes-removed-test.mdx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-02-07
+category: compute
+product: kubernetes
+---
+
+From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with cilium or `calico` as CNI. 
+To benefit from the newest minor versions, Scaleway recommends recreating clusters on one of the supported CNIs. Please find more details on our migration policies here.
```
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-test.mdx:13]:
```suggestion
To benefit from the newest minor versions, Scaleway recommends recreating clusters on one of the supported CNIs. Please find more details on [our migration policies here](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy/).
```

File: changelog/february2023/2023-02-07-kubernetes-removed-test.mdx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: Test
```
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-test.mdx:2]:
```suggestion
title: Container Network Interface (CNI) plugins Flannel and Weave are not supported any longer
```

File: changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+status: removed
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-02-07
+category: compute
+product: kubernetes
+---
+
+From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with cilium or `calico` as CNI.
```
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx:12]: From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with `cilium` or `calico` as CNI.

File: changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-02-07
+category: compute
+product: kubernetes
+---
+
+From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with cilium or `calico` as CNI. 
+To benefit from the newest minor versions, Scaleway recommends recreating clusters on one of the supported CNIs. Please find more details on our migration policies here.
```
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx:13]:
```suggestion
To benefit from the newest minor versions, Scaleway recommends recreating clusters on one of the supported CNIs. Please find more details on [our migration policies here](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy/).```

File: changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx (hunk 4 above)
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx:12]:
```suggestion
From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with `cilium` or `calico` as CNI. 
```

File: changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+---
+title: Test
```
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx:2]:
```suggestion
title: Container Network Interface (CNI) plugins Flannel and Weave are not supported any longer
```

**Review by farhakennar — APPROVED** [review]

File: changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx [hunk 7]
```diff
@@ -0,0 +1,14 @@
+---
+title: Container Network Interface (CNI) plugins Flannel and Weave are not supported any longer
```
Comment (ldecarvalho-doc) [inline changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx:2]:
```suggestion
title: Container Network Interface (CNI) plugins Flannel and Weave are no longer supported
```

File: changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+status: removed
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-02-07
+category: compute
+product: kubernetes
+---
+
+From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with `cilium` or `calico` as CNI.
```
Comment (bene2k1) [inline changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx:12]:
```suggestion
From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with `cilium` or `calico` as CNI.  
```

File: changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+author:
+    fullname: 'Join the #k8s channel on Slack.'
+    url: 'https://slack.scaleway.com'
+date: 2023-02-07
+category: compute
+product: kubernetes
+---
+
+From k8s version 1.25 onwards, all new clusters created in Scaleway Kubernetes will either be with `cilium` or `calico` as CNI. 
+To benefit from the newest minor versions, Scaleway recommends recreating clusters on one of the supported CNIs. Please find more details on [our migration policies here](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy/).```
```
Comment (ldecarvalho-doc) [inline changelog/february2023/2023-02-07-kubernetes-removed-flannel-weave.mdx:13]:
```suggestion
To benefit from the newest minor versions, we recommend you recreate your clusters on one of the supported CNIs. Please find more details on [our migration policies here](https://www.scaleway.com/en/docs/containers/kubernetes/reference-content/version-support-policy/).```
```

**Review by ldecarvalho-doc — APPROVED** [review]

**Pull request merged**